app/models/planning_application.rb

```diff
@@ -486,6 +486,12 @@ def recommendation
     @recommendation ||= recommendations.last
   end
 
+  def last_recommendation_accepted?
```

Comment:
I think we can use this [here](https://github.com/unboxed/bops/blob/main/app/models/assessment_details_review.rb#L105) as well 🎉

Reply:
I think it's covered in a later commit but I will double check.....